### PR TITLE
[TO FEATURE] DESENG-509: Unit test for Language API

### PR DESCRIPTION
**Pull request by ratheesh-aot**
Issue #:  https://apps.itsm.gov.bc.ca/jira/browse/DESENG-509
*Description of changes:*
     Added Unit test for Language API

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the met-public license (Apache 2.0).

### CODE REVIEW

File: met-api/tests/unit/api/test_language.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
+# Copyright © 2019 Province of British Columbia
```
Comment (Baelx) [inline met-api/tests/unit/api/test_language.py:1]: We checked with Saravan and there are no copyrights for this app. Could you please remove this section?

File: met-api/tests/utilities/factory_utils.py [hunk 2]
```diff
@@ -502,3 +504,14 @@ def factory_widget_map_model(widget_map: dict = TestWidgetMap.map1):
     )
     widget_map.save()
     return widget_map
+
+
+def factory_language_model(lang_info: dict = TestLanguageInfo.language1):
+    """Produce a Language  model."""
+    langauge = LanguageModel(
```
Comment (Baelx) [inline met-api/tests/utilities/factory_utils.py:511]:
```suggestion
    language = LanguageModel(
```

**Review by Baelx — CHANGES_REQUESTED** [review]
Great work! Could you please just remove the "Copyright" line. We won't be putting those in code anymore going forward. Also maybe just correct the spelling in one place but obviously that's minor!

File: met-api/tests/utilities/factory_utils.py (hunk 2 above)
Comment (ratheesh-aot) [reply met-api/tests/utilities/factory_utils.py]: Thanks, @Baelx for catching this.

**Review by Baelx — APPROVED** [review]

**Pull request merged**